documentation/src/main/asciidoc/mapping.asciidoc

```diff
@@ -1428,41 +1419,48 @@ full text representation.
 However, in some situation it can be useful to insert a custom token representing the `null` value.
 See <<field-annotation>> for more information.
 
-`java.lang.String`:: Strings are indexed as are
+`java.lang.String`:: Strings are indexed as they are.
+
+`boolean`, `Boolean`::
+Are converted into their string representation and indexed.
+
+`char`, `Character`::
+Are converted into their string representation and indexed.
 
-`short`, `Short`, `integer`, `Integer`, `long`, `Long`, `float`, `Float`, `double`, `Double`:: Are
-per default indexed numerically using a link:http://en.wikipedia.org/wiki/Trie[Trie structure]. You
-need to use a `NumericRangeQuery` to search for values. See also <<field-annotation>> and
-<<numeric-field-annotation>>
+`short`, `Short`, `integer`, `Integer`, `long`, `Long`, `float`, `Float`, `double`, `Double`::
+Are per default indexed numerically using a link:http://en.wikipedia.org/wiki/Trie[Trie structure] which
+allows for efficient range queries and sorting, resulting in better query response times.
+You need to use a `NumericRangeQuery` to search for values. See also <<field-annotation>> and
+<<numeric-field-annotation>>.
+
+[CAUTION]
+====
+Prior to Search 5, numeric field encoding was only chosen if explicitly requested via `@NumericField`.
+As of Search 5 this encoding is automatically chosen for the numeric types mentioned above.
```

Comment:
It was preexisting, but I think we should use `Hibernate Search` not `Search`.

Maybe a comma after `As of (Hibernate) Search 5` would be in order.